components/System/LocaleSwitcher.vue

```diff
@@ -37,7 +38,13 @@ export default Vue.extend({
   props: {
     fab: {
       type: Boolean,
-      required: true
+      required: false,
+      default: false
```

Comment:
This is unnecessary, since the default is always false (It works like HTML attributes)

Reply:
Done